functions.js

```diff
@@ -74,11 +70,32 @@ function clickLogic (clickedCard, deck) {
             divId : clickedCard.id,
             url : clickedCard.childNodes[0].src
         }
+        turnCounter++
+        document.getElementById('turnContainer').textContent = String(turnCounter)
```

Comment:
using `String()` is instantiating a String object and passing turn counter in. This is a relatively slow process, its quicker to use `toString()` like this: `turnCounter.toString()`